kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/core/v1/PodOperationsImpl.java

```diff
@@ -280,6 +278,30 @@ public Boolean evict() {
     }
   }
 
+  @Override
+  public Boolean evict(io.fabric8.kubernetes.api.model.policy.v1.Eviction eviction) {
```

Comment:
If we had a `private Boolean evict(HasMetadata eviction)` method, we could reuse most of the exception handling boiler-plate both for this method and the `evictThis` method.